rcl_interfaces/msg/ParameterValue.msg

```diff
@@ -6,8 +6,8 @@
 uint8 type
```

Comment:
> Hm, I guess we can't at the moment...

Should we open a ticket for this somewhere (rosidl?) that would be a useful feature